mathics/builtin/numbers/numbertheory.py

```diff
@@ -171,7 +216,8 @@ def apply(self, n, evaluation):
             factors = sympy.factorint(n.value)
             factors = sorted(factors.items())
             return Expression(
-                "List", *(Expression("List", factor, exp) for factor, exp in factors)
+                SymbolList,
+                *(Expression(SymbolList, factor, exp) for factor, exp in factors)
```

Comment:
element_conversion_fn probably could be used with a lambda closure here as well. 

I'll leave this for some other time though.